src/controllers/default.js

```diff
@@ -0,0 +1,17 @@
+// TODO confirm which version of UUID to use
+const uuid4 = require('uuid/v4')
+
+module.exports = function (request, reply) {
+  const context = {
+    pageTitle: 'Waste Permits'
+  }
+
+  // Generate a token
+  const token = uuid4()
+  console.log('Generated token: ' + token)
+
+  reply
+    .view('index', context)
+    .unstate('session')                   // Delete existing session cookie (if there is one)
```

Comment:
Fine for now, but we'll have to check this assumption that we would delete an existing session ID. In this particular case the fact they went back to the start shouldn't reset their session, but for now its nice to know how!

Possibly add a `TODO:` comment to check this assumption later for now, and we should work together to add a story around session management, once you've had a chance to look at the interaction with Dynamics.